node_modules/col-users/lib/api.js

```diff
@@ -196,3 +198,81 @@ var getOrCreateUser = module.exports.getOrCreateUser = function(canvasUserId, co
     }
   });
 };
+
+/**
+ * Get the users for the current course and their points
+ *
+ * @param  {Context}    ctx                             Standard context containing the current user and the current course
+ * @param  {Function}   callback                        Standard callback function
+ * @param  {Object}     callback.err                    An error that occurred, if any
+ * @param  {User[]}     callback.users                  The users in the current course
+ */
+var getUsers = module.exports.getUsers = function(ctx, callback) {
```

Comment:
As this is being used to get the engagement index which currently does things like sorting and searching in the UI, we are retrieving the full list. As the Collabosphere courses are expected to be small in number (35 - 50), this should be acceptable. If this ever becomes an issue, we can switch to paging and dedicated sorting and searching feeds.